### DISCUSSION TITLE
Modulo 1

### DISCUSSION THREAD

**Discussion by JorgeCanto3** [LaunchX-InnovaccionVirtual/CursoIntroPython#38] [Q&A] [answered]
Tengo una pequeña duda, en este ejercicio la razon por la cual se utiliza "str" es porque el parsec y los ligthyears ya tienen un valor asignado, a comparacion del ejercicio de la calculadora que no cuenta con valores y por eso se agrega el "int", no?

parsec = 11

lightyears = 0

print(str(parsec) + " parsec, is " + str(lightyears) + " lightyears")

-------------------------------------------------------------------------------
print("Calculadora")
first_number = input("Primer número: ")
second_number = input("Segundo número: ")
print(first_number + second_number)

print(int(first_number) + int(second_number)

y una duda mas, tenemos que crear un codigo como ejercicio en las partes que dice "Escribe tu codifo"?

Gracias, soy muy nuevo, disculpen si es muy sencillo lo que pregunto (╹ڡ╹ )

**Comment by Paulina-Ceciliano** [comment, marked as answer]
En el primer ejercicio se agrega el str porque es la manera de unirlo (o concatenar) las cadenas de texto sin que nos salga error, otra forma de unir datos es con comas ( , ) en vez de + y con esta forma no es necesario convertir los datos.
 En la calculadora se pone int (de integer) porque con el método input() todas las entradas de datos son tipo cadenas de texto (str), entonces para realizar sumas es necesario convertirlo a datos de tipo integer / entero, o float /decimal, en caso de querer realizar sumas decimales, de no ser así, cuando quieras hacer una suma te va a salir el texto junto, por ejemplo 2+2= 22 
Espero que te sea útil esta información compañero, recuerda que no hay dudas tontas, solo tontos que no preguntan :P
Éxito con tus katas!

**Comment by mgc14** [comment]
Hola compañeross, ¿Cada Kata se hace en una notebook nueva? 
¿O se hacen todas en la misma notebook?

**Reply by HectorHernandezT** [reply]
Puedes hacer un notebook nuevo por cada Kata y guardarlos en la misma carpeta. Puedes hacer uno para ejercicios de práctica de Documentación y otro para la Kata final. El objetivo es que practiques haciendo notebooks diferentes

**Reply by CuauPillado** [reply]
Te agradezco mucho la respuesta, me ayudaste bastante (aunque yo no lo haya preguntado).

**Reply by mgc14** [reply]
¡¡Gracias por el apoyo!!
